README.md

```diff
@@ -461,6 +501,9 @@ if (result.error) throw result.error;
 - `toplevel` (default `false`) - set to `true` if you wish to enable top level
   variable and function name mangling and to drop unused variables and functions.
 
+- `nameCache` (default `null`) - pass an object if you wish to cache mangled names
+  across multiple invocations of `minify()`.
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/2176/commits/95ae5a5ff6d07508a910b2001ea4d7c5dea4e146